CONTRIBUTING.md

```diff
@@ -16,6 +16,7 @@ When using MacOS, there are a couple of extra prerequisites that are required.
 
 ## Getting Started
 
+- :gear: Make sure [Corepack](https://nodejs.org/api/corepack.html) is enabled via `corepck enable`.
```

Comment:
```suggestion
- :gear: Run `corepack enable` to enable [Corepack](https://nodejs.org/api/corepack.html).
```

should probably streamline this a bit for better readability, there is also a typo in the command. We can remove this again if node enables corepack by default